server/player/player.go

```diff
@@ -1400,22 +1406,18 @@ func (p *Player) UseItem() {
 	}
 	switch usable := it.(type) {
 	case item.Chargeable:
-		useCtx := p.useContext()
-		if !p.usingItem {
-			if !usable.ReleaseCharge(p, p.tx, useCtx) {
-				// If the item was not charged yet, start charging.
-				p.usingSince, p.usingItem = time.Now(), true
-			}
-			p.handleUseContext(useCtx)
-			p.updateState()
+		if p.usingItem {
+			p.tryChargeItem(usable, false)
+			return
+		} else if time.Since(p.serverChargedItem) <= time.Millisecond*100 {
```

Comment:
Is there no way at all to distinguish the client wants to finish charging a crossbow vs. release it?